common/run_as_userid.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+parent: Common
+---
+
+# Run as UserID
+
+Run pyRACF commands as another UserID.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .experimental }
+> _This functionality is **Experimental** and is subject to major changes and even being removed entirely._

```

Comment:
We are considering this experimental?

Reply:
at the time i wrote up this doc it was hardly tested. Now that it is a bit more robust, i think this is appropriate.